stone/frontend/lexer.py

```diff
@@ -221,6 +223,17 @@ def t_ANY_NULL(self, token):
     def t_ANY_ID(self, token):
         r'[a-zA-Z_][a-zA-Z0-9_-]*'
         if token.value in self.KEYWORDS:
+            if (token.value == 'annotation_type') and self.cur_indent:
+                # annotation_type was added as a reserved keyword relatively
+                # late, when there was already an identifer with the same name
+                # in an internal spec. because annotation_type-the-keyword can
+                # only be used at the beginning of a non-indented line, this
+                # check lets both the keyword and the identifer coexist and
+                # maintains backward compatibility.
+                # TODO: this is kind of a hack, and we should get rid of it if
```

Comment:
agree with sentiment, but not sure focus on _internal_ spec makes sense in comment- I also wouldn't want to break external users of Stone if any happened to use `annotation_type`